stt/run_stt.py

```
@@ -88,544 +108,735 @@ def create_providers():
+    2. Fall back to Common Voice API
+    3. Fall back to hardcoded test.wav
+    
+    Returns: (audio_data, channels, sample_width, sample_rate, filename, duration_seconds, ground_truth)
+    """
+    print("Loading audio sample with fallback strategy...")
+    
+    # First try: Local files
+    try:
+        print("Attempting to load from local en_test folder...")
+        return load_local_sample(min_duration, max_duration, max_retries)
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)